pkg/api/norman/customization/setting/setting.go

```diff
@@ -70,21 +69,11 @@ func Validator(request *types.APIContext, schema *types.Schema, data map[string]
 		_, err = providerrefresh.ParseMaxAge(newValueString)
 	case "auth-user-info-resync-cron":
 		_, err = providerrefresh.ParseCron(newValueString)
-	case "kubeconfig-token-ttl-minutes":
+	case "kubeconfig-default-ttl-minutes":
```

Comment:
You can delete this validation altogether. We do not need to update it with the new setting. Currently, the UI does not use this validation since setting updates go through Steve, not Norman.